.github/workflows/push-to-azure-queue.yml

```diff
@@ -35,8 +36,18 @@ jobs:
     runs-on: ubuntu-latest
     needs: [ queue-pr ]
     steps:
-      - name: Create commit comment
-        uses: peter-evans/commit-comment@3f3629d9da4bf387b09fc75456a5b8a344ba5641 # pin@v2
+      - name: Create Comment on PR
+        uses: actions/github-script@7a5c598405937d486b0331594b5da2b14db670da # pin@v6
         with:
-          body: |
-            'This commit is queued for merging with the `test_next` branch! Please ignore this PR for now. Contact @microsoft/fluid-cr-infra for help.'
+          token: ${{ secrets.WORKFLOW_TOKEN }}
+          script: |
+            const { Octokit } = require("@octokit/core");
+            const pr = ${{ github.event.pull_request.number }}
+            const token = `${{ secrets.GITHUB_TOKEN }}`;
+            const octokit = new Octokit({ auth: token })
+            octokit.rest.pulls.createReviewComment({
+              owner: 'sonalivdeshpande',
```

Comment:
This needs to be configurable or come from the context of the workflow itself (maybe `${{ github.repository_owner }}`?